src/Components/NavSide/index.scss

```diff
@@ -27,30 +27,79 @@
 
   .menu {
     position: absolute;
-    bottom: $tagline-offset;
+    bottom: 1rem;
     background: none;
-    width: 100%;
     text-align: center;
-    // transform: rotate(-90deg);
 
     .menu-item {
-      @include hover-effect();
-
       color: white;
       margin-bottom: 1.25rem;
-      display: block;
-      margin-left: -.15em;
+      overflow: hidden;
+      text-align: left;
+
+      .menu-item__animation-wrapper {
+        align-items: center;
+        display: flex;
+        height: 2.5rem;
+        justify-content: flex-start;
+        margin-bottom: .3rem;
+        width: 2.5rem;
+      }
 
       &:last-child {
         margin-bottom: 0;
       }
+
+      &:hover {
+        .menu-item__animation-wrapper {
+          width: auto;
+        }
+        .menu-item__label {
+          transform: translate(0);
+        }
+
+        .menu-item__icon {
+          background: white;
+          color: $color-primary;
+        }
+      }
+
+      .menu-item__icon,
+      .menu-item__label {
+        box-sizing: border-box;
+        display: inline-block;
+        height: 100%;
+        position: relative;
+      }
+
+      .menu-item__icon {
+        padding: .5rem .25rem;
+        transition: all .2s ease-in-out;
+        width: 2.5rem;
+        z-index: 2;
+      }
+
+      .menu-item__label {
+        background: white;
+        box-shadow: 1px 1px 1px rgba(0,0,0,.2);
+        color: $color-primary;
+        font-size: .9rem;
+        line-height: 1.5rem;
+        padding: .5rem .75rem;
+        text-align: left;
+        text-transform: uppercase;
+        transform: translate(calc(-100% - 2.6rem));
+        transition: transform .2s ease-in-out;
+        transition-delay: .1s;
+        z-index: -1;
+      }
     }
 
     // chill with that beak poking out, bird
     .icon-twitter {
       font-size: .85em;
       // make it feel more centered
-      margin-left: -.07em;
```

Comment:
haha, thanks, I agree. I don't remember how it looked before that made me feel like the beak was poking out.